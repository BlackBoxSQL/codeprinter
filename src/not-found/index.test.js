import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './index.js';

describe('NotFound', () => {
  it('renders without crashing', () => {
    shallow(<NotFound />);
  });
});
