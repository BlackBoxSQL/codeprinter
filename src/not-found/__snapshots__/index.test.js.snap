// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div
  className="responsive-container container-fluid"
  style={
    Object {
      "marginTop": 10,
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h2>
        <i
          className="fas fa-map-signs text-warning"
        />
         404 Not Found
      </h2>
      <hr />
      <p>
        It looks like you're lost. Let's go 
        <a
          href="/"
          onClick={[Function]}
        >
          home
        </a>
        .
      </p>
    </div>
  </div>
</div>
`;
